diff --git a/convert/compile.sh b/convert/compile.sh
old mode 100644
new mode 100755
index 3c445f6..3c730df
--- a/convert/compile.sh
+++ b/convert/compile.sh
@@ -1,4 +1,4 @@
-#export BOOST=/path/to/boost
+export BOOST=/mnt/lustre/groups/nms_epapg/Hyper-K/software/boost/1_77_0
 
 g++  -O2 -Wall -fPIC -pthread -std=c++11 -m64 -I$BOOST -I$ROOTSYS/include -c main.cpp
 g++  -O2 -m64 -std=c++11 main.o -lm -L$ROOTSYS/lib -lCore -lRIO -lNet -lHist -lGraf -lGraf3d -lGpad -lTree -lRint -lPostscript -lMatrix -lPhysics -lMathCore -lThread -lMultiProc -pthread -lm -ldl -rdynamic -o waveconvert
diff --git a/scripts/Fit.C b/scripts/Fit.C
index 32bd949..788efec 100644
--- a/scripts/Fit.C
+++ b/scripts/Fit.C
@@ -242,7 +242,7 @@ void Fit(std::string inName){
 	std::cout << "resolution: " << resolution << std::endl;
 
 
-	TH1D *h1 = new TH1D("charge", "charge", 500, -50, 200); // define histogram 
+	TH1D *h1 = new TH1D("charge", "charge", 500, -50, 400); // define histogram 
 	h1->GetXaxis()->SetTitle("Integrated Voltage [mV]");
 
 	for (int i = 0; i < tWaves->GetEntries(); i++){ // fill histogram
@@ -324,8 +324,8 @@ void Fit(std::string inName){
 
 	pmt->SetParLimits(0,-3,3);
 	pmt->SetParLimits(1,0,30);
-	pmt->SetParLimits(2,0,40);
-	pmt->SetParLimits(3,0,10);
+	pmt->SetParLimits(2,0,170);//new digital, original is 40
+	pmt->SetParLimits(3,0,70);//new digital, original is 10
 	pmt->SetParLimits(4,0.,1);
 	pmt->SetParLimits(5,0.,1);
 	pmt->SetParLimits(6,0.001,1.5000);
@@ -354,7 +354,7 @@ void Fit(std::string inName){
 
 
 	double voltage = (pmt->GetParameter(2))*1e-3; 
-	double timeb = 4e-9; // time per bin
+	double timeb = 1e-9; // time per bin
 	double charge = timeb*voltage/50;
 	double e = 1.6e-19;
 	double gain = charge/e;
diff --git a/scripts/RunDarkSingle.sh b/scripts/RunDarkSingle.sh
index d8cffdb..b54dadc 100755
--- a/scripts/RunDarkSingle.sh
+++ b/scripts/RunDarkSingle.sh
@@ -8,7 +8,7 @@ if [ "${1: -4}" == "root"  ]
 			then
 				echo "Converted data exists."
 				echo ${nameroot}
-				root -l -b -q 'DarkRate.C("'${nameroot}'",5,1)'
+				root -l -b -q 'DarkRate.C("'${nameroot}'",12.2884,0.4)'
 		fi
 	else
 		echo "Wave data does not exist, skipping directory."
